tests/foreman/ui/test_syncplan.py

```diff
@@ -89,11 +89,15 @@ def test_positive_end_to_end(session):
     :expectedresults: All CRUD actions for component finished successfully
 
     :CaseLevel: Integration
+
+    :BZ: 1693795
     """
     plan_name = gen_string('alpha')
     description = gen_string('alpha')
     new_description = gen_string('alpha')
+    org = entities.Organization().create()
```

Comment:
Is this substantial for test itself (we require new organization for this specific test), or is it here to force access to entity inside `with session` block, so browser is actually started?

In latter case, I guess you can call `session.contenthost.read_all('current_user')` as workaround. It will open browser and read data that is already there on the screen, so you skip creating organization and selecting it. In my not rigorous test, `contenthost.read_all` was 10 seconds faster than create + select organization.

And yes, in hindsight, "open browser on access to first entity" was not that smart as I thought. There are at least two classes of legitimate tests that don't work well with this approach. Unfortunately, workaround above is best I can do at the moment.

Reply:
@mirekdlugosz  As you guessed, that org line was mostly just work around

I have changed to use module_org and added the line you suggested

thank you